### PR TITLE
enh(spring-properties): use a TreeMap rather than an HashMap to store properties  

### PR DESCRIPTION
**Pull request by murdos**
That way properties order is guaranteed and reproducible.

### CODE REVIEW

File: src/test/java/tech/jhipster/lite/generator/server/springboot/database/cassandra/domain/CassandraModuleFactoryTest.java [hunk 1]
```diff
@@ -76,16 +76,15 @@ void shouldBuildModule() {
       .and()
       .hasFile("src/main/resources/config/application.yml")
       .containing(
+        // language=yaml
```
Comment (renanfranca) [inline src/test/java/tech/jhipster/lite/generator/server/springboot/database/cassandra/domain/CassandraModuleFactoryTest.java:79]: I am curious. Is this a command to the prettier format?

**Review by renanfranca — APPROVED** [review]
Thank you for your work @murdos , nice improvements!  I like the commit prefixs, do you have any documentation about that?

File: src/test/java/tech/jhipster/lite/generator/server/springboot/database/cassandra/domain/CassandraModuleFactoryTest.java (hunk 1 above)
Comment (murdos) [reply src/test/java/tech/jhipster/lite/generator/server/springboot/database/cassandra/domain/CassandraModuleFactoryTest.java:79]: It works with IntelliJ and provides syntax highlighting :-) : https://www.jetbrains.com/help/idea/using-language-injections.html#use-language-injection-comments.

Comment (renanfranca) [reply src/test/java/tech/jhipster/lite/generator/server/springboot/database/cassandra/domain/CassandraModuleFactoryTest.java:79]: Ow! That's really nice!!! 😊 🫂

**Comment by murdos** [comment]
> Thank you for your work @murdos , nice improvements!  I like the commit prefixs, do you have any documentation about that?

Thanks 😁
Regarding commits, yeah, I try to follow [conventional commits](https://www.conventionalcommits.org/)

**Pull request merged**